### ISSUE TITLE
printing create invoice page vs invoice page

### ISSUE THREAD

**Issue by ImRobbieRobski** [jreinke/magento-admin-theme#4]
After using this template I noticed that when we print invoices (from the create invoice page) it is now printing on two pages, even with only 1 sku. This is due to larger margin and padding values on various aspects of the create invoice page compared to the default.

To be more specific the issue I have is when I print directly on the create invoice page after opening a new pending order and clicking the invoice button.

If you print an existing invoice, it will fit on one page (up to 2 skus), but this adds 2 or 3 more clicks plus load time on the site. It will actually print 4 (maybe 5) skus and the grand total, but the comments are pushed onto a 2nd page.

Also, when you print I know you can choose to print 1 page but it would be nice to not have to do this.

I would be satisfied if the create invoice page printed just like the regular invoice page. But as it is now the create invoice page is flawed because there are too many new margins that the default admin theme did not have.

1) At the very top above New Invoice for Order # There is a half inch of just blank empty space about twice as much as the default
2)The text box that contains the order number order date/status IP etc uses 1/4" above and below the text, this was smaller on the default. 
3) On the default template there is no border around around the header of each box and so it seems that there might be some extra padding here involved because of that
4) the padding inside of the billing and shipping address is about 1/4" above and below the address. On the default template it is about 1/8" 
5) Items to invoice... There is significant padding above and below each product, about 1/4" above the product and 1/8th below the product. On the default template it appears to be about 1/16" above and below
6) the empty row under the list of products is 3/8" tall, on default it is a little under 1/4"

Basically on the old invoice with just 1 item would print on a single page and have 2.5" until the end of the page.

I used a simple ruler to get the measurements not extremely precise but accurate enough to demonstrate my point.
